<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidgetClass</class>
 <widget class="QWidget" name="MainWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="videowidget" native="true"/>
   </item>
   <item>
    <widget class="QSlider" name="timeSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>10000000</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="timeLable">
        <property name="text">
         <string>未选定视频</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openfile">
        <property name="text">
         <string>打开文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playBtn">
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseBtn">
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopBtn">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fullBtn">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>全屏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>  音量： </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="volumeSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWidget.qrc"/>
 </resources>
 <connections/>
</ui>
